### ISSUE TITLE
Selectively show nullable DataMembers as optional

### ISSUE THREAD

**Issue by makotogitdev** [superstator/Swaggeratr#61]
By default Swaggeratr show nullable DataMembers as optional. This is kind of strange particularly in a read response contract.

**Comment by superstator** [comment]
What would a mandatory nullable parameter look like? Not that I'm opposed to an option to override that default, I'm just having trouble picturing the use case.

**Comment by makotogitdev** [comment]
I was just thinking it would be nice to be able to remove the "optional" text in data type description of a parameter within a contract. But I guess "optional" text can just imply that it is possible that the parameter can be null for read responses.

**Comment by superstator** [comment]
Yeah, I think given we're talking about JSON/XML datacontracts, "optional" should be read as "nullable" most of the time. Especially given part of the value with Swagger is the ability to generate code from those contracts. Imagine generating code according to a Swagger spec, expecting that a given value in a response is required and will always be present - if it's actually nullable, eventually a JSON response may leave it out, and that code might fail.
